recipe/meta.yaml

```
@@ -1,46 +1,87 @@
 {% set name = "pyFFTW" %}
 {% set version = "0.13.1" %}
-{% set sha256 = "09155e90a0c6d0c1f2d1f3668180a7de95fb9f83fef5137a112fb05978e87320" %}
+{% set sha256 = "ffc1b3e7f4f74ee38716d668d7606b5fbfded58343655c160c3644fc4acf5d55" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://github.com/{{ name }}/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
   sha256: {{ sha256 }}
   patches:
-    - no-system-include.patch
+    - patches/0001-cython3-updates-and-test-build-linker-flags.patch
+    - patches/0002-no-system-include.patch
 
 build:
   number: 0
-  skip: true  # [py<39 and arm64]
+  # fftw not available for s390x
+  skip: true  # [py<38 or py>311 or s390x]
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - cython                                 # [build_platform != target_platform]
-    - numpy                                  # [build_platform != target_platform]
     - {{ compiler('c') }}
+    - patch     # [not win]
+    - m2-patch  # [win]
   host:
-    - cython
+    - cython >=3
     - python
     - pip
-    - numpy
-    - fftw  # [win]
-    - fftw-static  # [not win]
+    - setuptools
+    - wheel
+    - numpy {{ numpy }}
+    - fftw 3.3.9
   run:
     - python
     - {{ pin_compatible('numpy') }}
+    - fftw 3.3.9  # [win]
```

```suggestion
    - fftw 3.3.9  # [win]
  run_constrained:
    - dask >=1.0
    - scipy >=1.8.0
```
https://github.com/pyFFTW/pyFFTW/blob/v0.13.1/setup.py#L758

---

ok